tests/typ/math/frac.typ

```diff
@@ -16,6 +16,10 @@ $ x = (-b plus.minus sqrt(b^2 - 4a c))/(2a) $
 // Test binomial.
 $ binom(circle, square) $
 
+---
+// Test multinomial coefficients.
+$ binom(n, k_1, k_2, k_3) $
```

Comment:
you also need to update the reference image for CI to pass. you can do it by passing the `--update` flag to the test runner.